Jenkinsfile.clean_old_issues

```
@@ -0,0 +1,49 @@
+pipeline {
+    agent any
+
+    triggers { cron('H 12 * * *') }
+
+    parameters {
+        string(name: 'ASSISTED_INSTALLER_DEPLOYMENT_REMOTE', defaultValue: 'https://github.com/openshift-assisted/assisted-installer-deployment', description: 'assisted-installer-deployment repo URL to use')
+        string(name: 'ASSISTED_INSTALLER_DEPLOYMENT_BRANCH', defaultValue: 'master', description: 'Branch on assisted-installer-deployment repo to use')
+        string(name: 'LOG_LEVEL', defaultValue: 'INFO', description: 'Log level')
+        string(name: 'DAYS', defaultValue: '180', description: 'Age from which tickets are considered old and will be addressed in days')
+        string(name: 'ISSUES_LIMIT', defaultValue: '50', description: 'Maximum number of issues to update')
+    }
+
+    environment {
+        JIRA_ACCESS_TOKEN = credentials('assisted-installer-bot-jira-access-token')
+        SLACK_TOKEN = credentials('slack-token')
+    }
+
+    stages {
+        stage('Init') {
+            steps {
+                sh "rm -rf assisted-installer-deployment"
+                sh "git clone ${ASSISTED_INSTALLER_DEPLOYMENT_REMOTE} --branch ${ASSISTED_INSTALLER_DEPLOYMENT_BRANCH}"
+            }
+        }
+
+        stage('Delete old issues') {
+            steps {
+                dir ('assisted-installer-deployment') {
+                    script {
+                        def integerDays = DAYS.toInteger()
+                        def integerIssuesLimit = ISSUES_LIMIT.toInteger()
+                        sh "python ./tools/triage/clean_old_tickets_attachments.py --days ${integerDays} --issues-limit ${integerIssuesLimit} --log-level ${LOG_LEVEL} --jira-access-token ${JIRA_ACCESS_TOKEN}"
```

why we need ``def integerDays = DAYS.toInteger()``? won't this just work?
```suggestion
                        sh "python ./tools/triage/clean_old_tickets_attachments.py --days ${DAYS} --issues-limit ${ISSUES_LIMIT} --log-level ${LOG_LEVEL} --jira-access-token ${JIRA_ACCESS_TOKEN}"
```

---

Yes it is redundant, it interpolates it anyway